3.3/examples/django/demo/settings.py

```diff
@@ -0,0 +1,116 @@
+"""
+Django settings for demo project.
+
+Generated by 'django-admin startproject' using Django 1.8.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.8/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.8/ref/settings/
+"""
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+import os
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.8/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '+u3anikga+6zkzk&@6l_rimzw$48p!2#52=$c2v#x#q2n@c0&e'
```

Comment:
@soltysh @rhcarvalho yeah, I think this is pretty natural to have in the template as parameter with generator that generates this value.... This value must be the same for all Pods created by RC right? So it must be an ENV var we set from top.